src/engine/source/builder/builders/opBuilderHelperMap.cpp

```diff
@@ -229,7 +229,7 @@ opBuilderHelperIntTransformation(const std::string& targetField,
 
         default:
             throw std::runtime_error(
-                fmt::format("\"{}\" function: Invalid parameter type of \"{}\"",
+                fmt::format(R"("{}" function: Invalid parameter type of "{}")",
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-%28C-plus-plus%29#modern-language-features-and-best-practices

Raw strings